TU20Bot/Models/CSVData.cs

```diff
@@ -0,0 +1,18 @@
+﻿using CsvHelper.Configuration.Attributes;
+
+namespace TU20Bot.Configuration {
+
```

Comment:
This is personal preference but... I'd rather you don't put a space at the beginning of every class and method. I don't think it serves a purpose, and the rest of the project doesn't have this. It should be tightly packed:

```
class MyClass {
  int a;
  int b;

  void x() {

  } // a nice waterfall of } at the end here too
}
```